### DISCUSSION TITLE
Ai最難取代的10種行業

### DISCUSSION THREAD

**Discussion by wu51000** [1101-GS4719-C/Python-Community-GS4719-C#1121] [General]
雖然可能沒有很準
但大家還是可以看一下自己有沒有機會在上面
有的話還至少安心一點（？？
不然的話祝大家都可以努力讀到博士當教授～
https://www.gvm.com.tw/article/46062

**Comment by thomasxshen** [comment]
在人工智慧(AI)的時代，有些行業被認為是AI較難取代的。這些行業通常涉及高度的創造力、人際互動、情感判斷或複雜的策略決策。例如，教育工作者需要根據學生的不同特點和需求制定個性化的教育方案，並關注學生的學習進展和情感健康。醫療專業人員，如醫生和護士，則需要進行全面的病情評估和診斷，提供個性化的治療方案，並關注病人的身體和情感健康。律師需要深入理解法律制度和司法程序，並根據個案情況制定辯護策略。創意行業的專業人士，如藝術家、作家、音樂家、編劇和導演，則需要高度的創造力和想象力，並與觀眾或聽眾建立情感聯繫。這些行業的共同點在於，它們都需要人類獨特的情感和創造力，這是目前AI技術所無法複製的。因此，即使在AI技術不斷進步的今天，這些行業仍然被視為相對安全，不易被AI取代。

**Comment by yxin214** [comment]
在點進文章前我先猜了是演員、藝術表演者和運動員等，結果居然一個都沒上榜...。舉例來說，不會有人想看一群機器人比賽跑步吧，透過訓練一步步突破人體的極限，我認為是運動員的精隨所在。

**Comment by wanka2030** [comment]
老師的工作不單單只是要教育不同年紀的小孩之外，對於每位學生都應該充滿不同的教學熱忱以及教學方式，但大多數的開發者都希望老師能構成違背AI取代的一部份，內文寫到老師需要面臨不同的學生之外，管理活生生的生命應該也還輪不到AI來管理吧(?

**Comment by LinChloe** [comment]
心理輔導員?
感覺如果是AI當的話只會被嗆說:你又不是人類怎麼會了解我的想法
還有牧師，修女，教皇之類跟宗教相關的也很難。
不知如果未來AI有意識會不會想創一個AI教。
如果有的話我想加入進去。

**Comment by YifenLin** [comment]
其我在點進去文章以前，我自己心中覺得榜上有名會是作家或是創作者ㄟ(Maybe ytr, 喜劇演員...)，可能因為生程式AI的發展，導致這些內容創作者沒有在榜上吧。不過我的想法是畢竟AI的演算法是根據他接收的內容進行修改，所以目前生成出的內容原創性還是會有限(這我的想法歡迎大家討論~。不過也許在不久的將來，生程式AI生成出的內容原創性會有大突破也不一定。

**Comment by jylin34** [comment]
我覺得就現在來看，距離ai真的取代工作還有很長一段距離，就算ai 訓練的在怎麼厲害，他能做的還是要他"學習"過的才做得出來，如果在工作上，遇到一個突發事件，是以前完全沒發生過的，ai就有可能無法應對。所以我能做的就是活在當下，繼續學習新知，不要擔心未來有可能被ai取代的事情，否則會每天活在憂慮之中。

**Comment by Hahahahbla** [comment]
那些需要高度創造力、情感智能和人際互動的工作往往更難被人工智能取代。例如，藝術家、作家、音樂家、設計師等創意職業，以及醫生、心理諮詢師、教師等需要情感連結和人性化關懷的工作，都屬於難以被 AI 取代的範疇。

**Comment by yyujimmy** [comment]
醫生和護士
律師和法官
老師和教育工作者
記者和編輯
軍人和警察
政府官員和公共行政人員
職業運動員和教練
心理學家和心理諮詢師
演員和表演藝術工作者
藝術家和設計師
這些行業的特點在於它們需要高度的人際交往能力、創造力、情感智慧以及道德判斷力，這些是目前AI技術難以完全模擬和替代的。所以，如果你在這些行業中，或者打算進入其中之一，可以為未來的就業安全感到一點點安心。當然，學術界的博士教授也是一個不錯的選擇，因為知識創造和傳授永遠是人類社會中不可或缺的一部分！

**Comment by William08290829** [comment]
我看到網站有寫到小說家，但我感覺未來AI成為小說家也不是什麼問題吧。雖然不可能會寫的比人類更有創意或是有趣，但以正常水平的小說家來說感覺AI是還算可以勝任的。

**Comment by jeff110303549** [comment]
我同意這些行業雖然可能受到科技的影響，但仍然具有人類獨特的特質和價值，因此不太容易被AI取代。這些行業中的工作通常涉及到情感智慧、創造力、人際互動和專業知識，這些是目前的人工智慧所難以完全模擬的。因此，雖然我們可以看到AI在許多領域的進步，但仍然有許多工作是需要人類的參與和貢獻的。

我認為人工智慧的發展為我們帶來了許多機遇和挑戰，但人類的價值和專業知識仍然是不可或缺的。因此，我們應該持開放的態度，積極運用科技，同時保持對人文價值和人類專業的尊重和重視。

**Comment by jim1743** [comment]
我認為遊戲製作也是很難被AI取代的行業，因為遊戲的目標群眾是人類，然而每個人的喜好都不會相同，以任天堂這間公司為例，他們遊戲的開發目標是設計出讓大人小孩老人都能玩得開心的遊戲，並且遊戲的內容必須要新奇有趣，之前看到，任天堂公司的每個員工必須至少一項工作外的興趣，為的是讓員工多一項設計遊戲的靈感，因此我不認為AI有辦法創作遊戲。

**Comment by sandyho0307** [comment]
如文章所述，AI時代並不是所有職業都會面臨消失的風險，有些領域可能會因為AI的幫助而擴展新的職業角色，特別是在醫療、教育和科技創新領域。因此，我覺得未來的關鍵在於如何與機器更好地協同工作。

**Comment by cassy0201** [comment]
我覺得涉及創意發想、人際互動、互動交流的工作較不容易被取代

**Comment by OHoln** [comment]
我認為比起擔心自己會不會被取代，持續進步跟上時代的潮流才是上策，畢竟誰也無法預測為來，十年前的今天恐怕也沒人能預料到現在的AI發展如此之驚人吧。

**Comment by 710388** [comment]
還好吧，我比較想探討ai輔助下十大崛起的產業

**Comment by YeeDaniel** [comment]
創意產業：包括藝術家、作家、音樂家和設計師，這些職業需要創意和獨特的視角。
心理治療師和諮詢師：這些職業需要高度的情感理解和人際互動技能。
教師：特別是幼兒和初等教育教師，需具備情感支持和個性化指導能力。
醫生和護理人員：尤其是在提供個性化護理和複雜診斷方面，需要高度的專業知識和同理心。

**Comment by sunset513** [comment]
救我來看， ai 就像工業革命一樣，必定會取代許多傳統的工作。但在這個同時，必定也會創造更多的機會，我們一定要緊密觀察，才能抓住時代的潮流。

另外，我覺得我們不要把注意力一直放在害怕自己被取代，這樣可能會過度焦慮。我們更應該把注意力放在該如何跟上時代，這樣才能夠真正避免自己被淘汰。

**Comment by yubo0oo** [comment]
我認為與其擔心會被淘汰
不如跟上時代改變
與時俱進才是不被淘汰的真理
